packages/untp-playground/src/lib/schemaValidation.ts

```diff
@@ -18,21 +19,117 @@ const SCHEMA_URLS = {
   DigitalIdentityAnchor: 'https://test.uncefact.org/vocabulary/untp/dia/untp-dia-schema-0.2.1.json',
 };
 
+const EXTENSION_VERSIONS: Record<
+  string,
+  { domain: string; versions: { version: string; core: { type: string; version: string } }[] }
+> = {
+  DigitalLivestockPassport: {
+    domain: 'aatp.foodagility.com',
+    versions: [
+      {
+        version: '0.4.0',
+        core: { type: 'DigitalProductPassport', version: '0.5.0' },
+      },
+    ],
+  },
+};
+
+const schemaURLConstructor = (type: string, version: string) => {
+  const shortCredentialTypes: Record<string, string> = {
+    DigitalProductPassport: 'dpp',
+    DigitalConformityCredential: 'dcc',
+    DigitalTraceabilityEvent: 'dte',
+    DigitalFacilityRecord: 'dfr',
+    DigitalIdentityAnchor: 'dia',
+  };
+  return `https://test.uncefact.org/vocabulary/untp/${shortCredentialTypes[type]}/untp-${shortCredentialTypes[type]}-schema-${version}.json`;
+};
+
+const extensionSchemaURLConstructor = (type: string, version: string) => {
```

Comment:
Should we hardcode this extension? Do we have any ways to handle value more flexibly?

Reply:
The schema URL is constructed from the UNTP credential type and version. So I think it's flexible.